src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -65,6 +65,23 @@ private function hasDefaultValue(\ReflectionParameter $parameter)
     }
 
     /**
+     * Returns if the argument is allowed to be null but is still mandatory.
+     *
+     * @param \ReflectionParameter $parameter
+     *
+     * @return bool
+     */
+    private function isNullable(\ReflectionParameter $parameter)
+    {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
I've been poking around on `#hhvm` and basically what I was told:
> HHVM recommends doing feature detection, not version sniffing

I would really love to see a generic feature detection system which is determined either during container compilation or kernel boot-up